guides/common/modules/ref_best-practices-for-activation-keys.adoc

```diff
@@ -0,0 +1,5 @@
+[id="best-practices-for-activation-keys_{context}"]
+= Best practices for activation keys
+
+* {Team} recommends using a single activation key per use case.
```

Comment:
```suggestion
* Create an activation key for each use case.
Sentence explaining why this is a good idea.
```
We tend not to use "{Team} recommends" in d/s docs. Now we just say, "Do this", and tell them why, even if it's really obvious, like "use meaningful names so that you can figure out what your keys are for" :-)